EZSwiftExtensionsTests/ArrayTests.swift

```diff
@@ -126,13 +126,14 @@ class ArrayTests: XCTestCase {
         XCTAssertEqual(difference, [0, 5, 6, 7, 8])
     }
 
+    /// WARNING: [SE-0121] Remove Optional Comparison Operators
     func testOptionalEquatable() {
         let a: [Int]? = [1, 2, 3]
```

Comment:
Variants of == and != which accept optional operands are still useful, and their results unsurprising, so they will remain.

I think this is useful